cabra/src/pages/components/MiniTaskDelete.tsx

```diff
@@ -0,0 +1,32 @@
+import "twin.macro";
+
+import NavigationButton from "./NavigationButton";
+import { TrashIcon } from "@heroicons/react/solid";
+
+interface Props {
+  title: string;
+  relationId: string;
+  taskId: string;
+  onClickDelete: CallableFunction;
```

Comment:
You can be more specific with typing here
```suggestion
  onClickDelete: (relationId: string, taskId:string) => void ;
```